### PR TITLE
Upload EKS-A release manifest as part of weekly release

### PR DESCRIPTION
**Pull request by abhay-krishna**
This PR includes the EKS-A releases manifest in the weekly release assets.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ DRY_RUN=false
 endif
 DATE_YYYYMMDD=$(shell date "+%F")
 WEEKLY?=false
-WEEKLY_BUNDLE_MANIFEST_URL=https://dev-release-assets.eks-anywhere.model-rocket.aws.dev/weekly-releases/bundles/$(DATE_YYYYMMDD)/manifest.yaml
+WEEKLY_RELEASES_URL_PREFIX=https://dev-release-assets.eks-anywhere.model-rocket.aws.dev/weekly-releases
```
Comment (jaxesn) [inline release/Makefile:33]: Didn't think about it before, shouldn't this be on the staging account/bucket?

**Review by jaxesn — APPROVED** [review]
Seems reasonable.

File: release/Makefile (hunk 1 above)
Comment (abhay-krishna) [reply release/Makefile:33]: I wanted to keep the staging account as an intermediate between the dev and prod, kind of as an interim location for prod release candidates until they've been qualified for release. This weekly release I see as an "official" dev release so I feel it's more at home in this location.

**Comment by abhinavmpandey08** [comment]
> @abhay-krishna: you cannot LGTM your own PR.

😡 😤

**Review by abhinavmpandey08 — APPROVED** [review]
/approve

**Pull request merged**